ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_histograms.py

```diff
@@ -0,0 +1,32 @@
+import ivy_tests.test_ivy.helpers as helpers
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+@handle_frontend_test(
+    fn_tree="numpy.bincount",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
+    ),
+    shape=helpers.get_shape(),
```

Comment:
This is throwing an error
`E   hypothesis.errors.InvalidArgument: test_numpy_bincount() got an unexpected keyword argument 'shape', from `shape=get_shape()` in @given` because it is never used anywhere in the tests.